### PR TITLE
Google combine move remove

### PR DESCRIPTION
**Pull request by wrp801**
Combined the following into one branch: 
- sc-3330/google-cloud-storage-remove-file-s
- sc-3332/google-cloud-storage-move-file-s

### CODE REVIEW

File: googlecloudstorage_blueprints/move_file.py [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+    """
+    Return a list of all file_names that matched the regular expression.
+    """
+    matching_file_names = []
+    for blob in file_blobs:
+        if re.search(file_name_re, blob.name):
+            matching_file_names.append(blob)
+
+    return matching_file_names
+
```
Comment (blakeburch) [inline googlecloudstorage_blueprints/move_file.py]: For sustainability, I'd prefer to remove reference to this function and use files.find_all_file_matches() from shipyard_utils. It does exactly the same thing, with the addition of printing the number of matches and their names (which means some of your current prints might need to be rearranged).

Want to centralize this so if the logic ever has to be updated, it gets updated in every Blueprint.

File: googlecloudstorage_blueprints/remove_file.py [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+def gcp_find_matching_files(file_blobs, file_name_re):
+    """
+    Return a list of all file_names that matched the regular expression.
+    """
+    matching_file_names = []
+    for blob in file_blobs:
+        if re.search(file_name_re, blob.name):
+            matching_file_names.append(blob)
+
+    return matching_file_names
```
Comment (blakeburch) [inline googlecloudstorage_blueprints/remove_file.py:135]: For sustainability, I'd prefer to remove reference to this function and use files.find_all_file_matches() from shipyard_utils. It does exactly the same thing, with the addition of printing the number of matches and their names (which means some of your current prints might need to be rearranged).

Want to centralize this so if the logic ever has to be updated, it gets updated in every Blueprint.

**Review by blakeburch — APPROVED** [review]
Small change requested to use Shipyard Utils function. This shouldn't block and you can merge once tested.

Everything looks good functionality wise.

**Comment by wrp801** [comment]
Just replaced the the code for finding all matching files, if you’d like to take a look. I just successfully tested locally and in Shipyard

**Pull request merged**